microshift_getting_started/microshift-understanding.adoc

```diff
@@ -6,6 +6,6 @@ include::_attributes/attributes-microshift.adoc[]
 
 toc::[]
 
-Get an overview of what you can do with {product-title}, based on {OCP}, but designed for optimizing small form factor devices and edge computing.
+Get an overview of what you can do with {product-title}, a Kubernetes distribution based on {OCP} that is designed for optimizing small form factor devices and edge computing.
```

Comment:
we standarized on "**derived**" from OCP

Reply:
yes we did, oversight